src/libsync/httplogger.cpp

```diff
@@ -61,7 +61,8 @@ void logHttp(const QByteArray &verb, const QString &url, const QByteArray &id, c
         if (reply->attribute(QNetworkRequest::HttpPipeliningWasUsedAttribute).toBool()) {
             stream << "Piplined,";
         }
-        stream << duration << ")";
+        const auto int_ms = std::chrono::duration_cast<std::chrono::milliseconds>(duration);
+        stream << duration << "=" << int_ms.count() << "ms)";
```

Comment:
```suggestion
        const auto durationInMs = std::chrono::duration_cast<std::chrono::milliseconds>(duration);
        stream << durationInMs.count() << "ms)";
```

Logging the duration once is more than sufficient.